rxdogtag/src/main/java/rxdogtag2/RxDogTag.java

```diff
@@ -518,9 +517,10 @@ public void install() {
     /** Default ignored packages when sourcing originating subscribe points. */
     private static final Collection<String> DEFAULT_IGNORED_PACKAGES =
         asList(
-            // "io.reactivex"
-            Observable.class.getPackage().getName(),
-            // "com.uber.rxdogtag"
+            // RxJava 3 package name. Referencing by type would have been nice but
+            // io.reactivex.rxjava3 doesn't have any base classes.
+            "io.reactivex.rxjava3",
```

Comment:
Observable.class.getPackage would now return `io.reactivex.rxjava3.core`. There are no classes in `io.reactivex.rxjava3` so we would have to refer to this loosely typed String